src/View/Helper/HtmlHelper.php

```diff
@@ -344,7 +344,11 @@ public function link($title, $url = null, array $options = [])
             $title = h(urldecode($title));
             $escapeTitle = false;
         }
-
+        
```

Comment:
@antograssiot Could you please check if we can add a sniff for trailing whitespace to our ruleset?